identity/identity.go

```diff
@@ -35,10 +37,11 @@ type (
 		// Credentials represents all credentials that can be used for authenticating this identity.
 		Credentials map[CredentialsType]Credentials `json:"-" faker:"-" db:"-"`
 
-		// TraitsSchemaURL is the JSON Schema to be used for validating the identity's traits.
-		//
-		// format: uri
-		TraitsSchemaURL string `json:"traits_schema_url,omitempty" faker:"-" db:"traits_schema_url"`
+		// TraitsSchemaID is the JSON Schema to be used for validating the identity's traits.
+		TraitsSchemaID string `json:"traits_schema_id" faker:"-" db:"traits_schema_id"`
+
+		// TraitsSchemaURL is the kratos public endpoint where the identity's traits schema can be fetched.
+		TraitsSchemaURL string `json:"traits_schema_url" faker:"-" db:"-"`
```

Comment:
Add `format: uri`

Reply:
Uniform Resource Locator (URL)
Uniform Resource Identifier (URI)
Uniform Resource Name (URN)

as we need the location it should be `url` I think